upgrade/breaking-changes/3-0-0.md

```diff
@@ -25,8 +25,8 @@ Grid EditorTemplates and Keyboard Navigation are in design stage. So, there migh
 
 ## Common Changes
 
-- [`IconName` class]({%slug common-kb-migration-from-iconname%}) (already obsolete) class will be removed. All icon names should be retrieved from the [built-in icons documentation]({%slug general-information/font-icons%}#icons-list).
-- `ToODataString` extension method is moved to the `Telerik.Blazor.Extensions` namespace.
+- [`Telerik.Blazor.IconName` class]({%slug common-kb-migration-from-iconname%}) (already [obsolete since version 2.0]({%slug changes-in-2-0-0%})) is removed. Use the icon names from the [Built-in Icons documentation]({%slug general-information/font-icons%}#icons-list).
```

Comment:
```suggestion
- [`Telerik.Blazor.IconName` class]({%slug common-kb-migration-from-iconname%}) ([obsolete since version 2.0]({%slug changes-in-2-0-0%})) is removed. Use the icon names from the [Built-in Icons documentation]({%slug general-information/font-icons%}#icons-list).
```